src/react/features/tools/update/fields.js

```diff
@@ -3,6 +3,7 @@ export default [
     key: 'text',
     label: 'Text',
     format: 'string',
+    autoFocus: true,
```

Comment:
Good point, didn't realise that this is a property of the field and field properties are defined here.

And thanks for making the change for me!